### DISCUSSION TITLE
v1.33.1

### DISCUSSION THREAD

**Discussion by mikecao** [umami-software/umami#1239] [General]
## Minor update

The `check-db` script will now automatically apply database changes for you. This is to provide a smoother onboarding experience for users, especially for those running Docker.

<hr /><em>This discussion was created from the release <a href='https://github.com/mikecao/umami/releases/tag/v1.33.1'>v1.33.1</a>.</em>

**Comment by TS263727** [comment]
能不能增加一个访问ip地址，显示地区那种
